README.md

```diff
@@ -17,15 +17,15 @@
 A JupyterLab extension for [conda-store][conda-store-repo].
 
 > [!NOTE]
-> Currently, this extension is only compatible with JupyterLab `>= 3.0 <= 4.0`.
+> This extension is only compatible with JupyterLab `>= 4.0`.
```

Comment:
To maintainers: after releasing a new version I would update this to say something like "Version YYYY.MM.X+ is only compatible with JupyterLab `>= 4.0, use version YYYY.MM.Y for older JupyterLab".